lib/TfIdfVectorizer.jsm

```diff
@@ -0,0 +1,74 @@
+
+const REGEXP_SPLITS = new RegExp(`[${UNICODE_SPACE}${UNICODE_SYMBOL}${UNICODE_PUNCT}]+`);
+const REGEXP_ALPHANUMS = new RegExp(`^[${UNICODE_NUMBER}${UNICODE_MARK}${UNICODE_LETTER}]+$`);
+
+this.TfIdfVectorizer = class TfIdfVectorizer {
+  /**
+   * Downcases the text, and splits it into consecutive alphanumeric characters.
+   * This is locale aware, and so will not strip accents. This uses "word
+   * breaks", and os is not appropriate for languages without them
```

Comment:
what do you mean by "os"?

Reply:
That's just a typo. It looks like I accidentally doubled the word "is", but ended up typing it wrong the first time.